proto/alarm/v1/stream.proto

```diff
@@ -45,28 +45,47 @@ message AlarmLogEntry {
 
     string chart = 3;
     string name = 4;
-    uint64 batch_id = 5;
-    uint64 sequence_id = 6;
-    uint64 when = 7;
+    string family = 5;
+    uint64 batch_id = 6;
+    uint64 sequence_id = 7;
+    uint64 when = 8;
 
-    string config_hash = 8;
+    string config_hash = 9;
```

Comment:
note: changing the id of the field 8->9 is not a backward compatible change. It is ok since we haven't gone live with those messages yet.

Reply:
Yes, thanks Leonida.